docs/lib/project-setup/fragments/android/coroutines/coroutines.md

```diff
@@ -0,0 +1,125 @@
+Amplify provides an optional and separate API surface which is entirely focused on using Kotlin's [coroutines](https://developer.android.com/kotlin/coroutines?gclid=CjwKCAiAmrOBBhA0EiwArn3mfF_RLMKM4EoDx1LThVJsHbeJ4fwEYJuFJ6wr5WS35qbasipfG3g-GhoCFQsQAvD_BwE&gclsrc=aw.ds) and [flows](https://developer.android.com/kotlin/flow).
```

Comment:
Is this long query string needed? I think you just need .../coroutines

Reply:
Oh thanks! Good catch. Looks like a client tracking ID. Will remove.